generator/pubspec.yaml

```diff
@@ -22,3 +22,8 @@ dependencies:
 dev_dependencies:
   test: 1.17.4
   source_gen_test: ^1.0.0
+
+dependency_overrides:
+  retrofit:
```

Comment:
Please DO NOT commit these lines. It can break the package publishment.

Reply:
I add cacheControl in annotation
the generator should be dependency new annotation version
i fix this commit but build will be faild